dask/base.py

```diff
@@ -139,11 +141,20 @@ def normalize_function(func):
 
 
 normalize_token = Dispatch()
-normalize_token.register((int, float, str, tuple, list), lambda a: a)
-normalize_token.register(object,
-        lambda a: normalize_function(a) if callable(a) else a)
+normalize_token.register((int, float, str, unicode, bytes, tuple, list,
```

Comment:
At the very least, I would recourse for tuple and dict. These are the standard ways to serialize function arguments.